### PR TITLE
Add vendor table

### PR DESCRIPTION
**Pull request by edeyoung**
To test:

* Ensure that endpoints are mapped to vendors:

```
select url, vendor_name, http_response from endpoint_export;
```

* Ensure that products are mapped to vendors:

```
select hitp.name as product_name, vendor.name as vendor_name from healthit_products as hitp, vendors as vendor where hitp.vendor_id = vendor.id limit 50;
```

---

Pull requests into this repository require the following checks to be completed by the submitter and reviewers.

**Submitter:**
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR.
  - The JIRA ticket for this PR is: https://oncprojectracking.healthit.gov/support/browse/LANTERN-180
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.

**Primary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

**Secondary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code accomplishes the tasks purpose.
- [x] Tests are complete.
  - Tests have been run locally and pass.

**Merger:**

Prior to merging, ensure that all checks have been completed. Delete the branch after the merge
is complete.

Create a new branch `<branch name>_gomodupdate`. Run `make update_mods` to update the `go.mod`
and `go.sum` files to point to master. Create a PR. Require one sanity check reviewer.

### CODE REVIEW

File: endpointmanager/pkg/chplquerier/vendorquerier_test.go [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+}
+
+func Test_convertVendorJSONToObj(t *testing.T) {
+	var ctx context.Context
+	var err error
+
+	// basic test
+
+	vendorListJSON := `{
+		"developers": [
```
Comment (mayerm94) [inline endpointmanager/pkg/chplquerier/vendorquerier_test.go:115]: Any reason why endpointmanager/pkg/chplquerier/testdata/chpl_vendors.json can't be used for this?

File: endpointmanager/pkg/chplquerier/vendorquerier_integration_test.go [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+	vend.Status = "other"
+	err = persistVendor(ctx, store, &chplVend)
+	th.Assert(t, err == nil, err)
+
+	err = ctStmt.QueryRow().Scan(&ct)
+	th.Assert(t, err == nil, err)
+	th.Assert(t, ct == 1, "did not store data as expected")
+	storedVend, err = store.GetVendorUsingName(ctx, "Epic Systems Corporation")
+	th.Assert(t, err == nil, err)
+	th.Assert(t, vend.Equal(storedVend), "2stored data does not equal expected store data")
```
Comment (bmath10) [inline endpointmanager/pkg/chplquerier/vendorquerier_integration_test.go:70]: This error message and the one above it should probably be changed to something like `"initial stored data does not equal expected store data"` and `"updated stored data does not equal expected store data"` or something.

File: endpointmanager/pkg/chplquerier/vendorquerier_test.go (hunk 1 above)
Comment (edeyoung) [reply endpointmanager/pkg/chplquerier/vendorquerier_test.go:115]: mostly to compare against the expected JSON so I don't need to have a much larger comparison list.
